ScratchDjango/User/views.py

```diff
@@ -0,0 +1,46 @@
+import json
+
+import requests
+from django.contrib import messages
+from django.shortcuts import render
+from rest_framework import status
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from .forms import UserForm
+from .models import User
+
+HOST = "http://127.0.0.1:8000"
+
+
+# API Views
+@api_view(["POST"])
+def register_user(request):
+    data = request.data
+    try:
+        user = User.objects.create_user(password=data["password"], email=data["email"])
+        return Response({"Success": "User Registered"}, status=status.HTTP_200_OK)
+    except Exception as e:
+        return Response({"Failed": str(e)}, status=status.HTTP_400_BAD_REQUEST)
+
+
+# Template Views
+def register_user_template(request):
+    if request.method == "POST":
+        email = request.POST["email"]
+        password = request.POST["password"]
+        url = f"{HOST}/user/register_user/"
+        payload = json.dumps({"email": email, "password": password})
```

Comment:
It would be better to call API from the templates

Reply:
Oh using Javascript?

Reply:
Will have to change every feature since all the features are implemented like this. Is this necessary ?